.github/CODEOWNERS

```diff
@@ -93,3 +93,6 @@ api-report/undo-redo.api.md                                               @micro
 api-report/view-adapters.api.md                                           @microsoft/fluid-cr-framework
 api-report/view-interfaces.api.md                                         @microsoft/fluid-cr-framework
 api-report/web-code-loader.api.md                                         @microsoft/fluid-cr-loader
+
+# No owners for package.json files to avoid review spam when mono-repo dependencies are updated
+package.json
```

Comment:
other changes to package.jsons should be relevant, but it's probably true enough that changes that only touch a package's package.json are almost always infra-related